wordpress-mobile/WordPress-iOS
# Issues/6026 noticon update

Fixes #6026 

Of interest, Xcode 8 introduces some changes in the xib format.  After updating, for some reason I couldn't fathom the starting width of the noticon container view in `awakeFromNib` was 1000, which was creating a ridiculously large corner radius and preventing the white border from being rendered properly.  Changed this to use a const like the noticon view. 

To test: 
Launch the app and check notifications.  
Confirm that layout of noticon labels is correct.
Test a vault press notification and ensure the correct noticon is displayed.

Needs review: @jleandroperez
